### ISSUE TITLE
Feature Request: Advanced Alloy Smelter 

### ISSUE THREAD

**Issue by Keridos** [HenryLoenwind/EnderIOAddons#20]
Having a better Alloy Smelter would be nice, something that makes alloy smelting faster, but also more expensive power/resource wise. Maybe upgrading an alloy smelter with a rather expensive recipe to the advanced alloy Smelter which may be faster (configurable in config would be nice) than the basic one. I'd say increase power usage either exponentially or double as fast as the speed (e.g 4x power usage for 2x speed).

**Comment by Mementh** [comment]
agreed.. without mekanism in a pack smelting alot takes longer then i wish :(
